apps/docs/pages/guides/ai/choosing-compute-addon.mdx

```diff
@@ -21,6 +21,36 @@ The number of dimensions in your embeddings is the most important factor in choo
 
 ## HNSW
 
+### 384 Dimensions
+
+This benchmark uses the dbpedia-entities-openai-1M dataset, which contains 1,000,000 embeddings of text. Each embedding is 384 dimensions created with the [gte-small](https://huggingface.co/Supabase/gte-small).
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="gte384"
+>
+<TabPanel id="gte384" label="gte-small-384">
+
+| Plan   | Vectors   | m   | ef_construction | ef_search | QPS  | Latency Mean | Latency p95 | RAM Usage  | RAM    |
+| ------ | --------- | --- | --------------- | --------- | ---- | ------------ | ----------- | ---------- | ------ |
+| Free   | 100,000   | 16  | 64              | 60        | 580  | 0.017 sec    | 0.024 sec   | 1.2 (Swap) | 1 GB   |
```

Comment:
```suggestion
| Starter   | 100,000   | 16  | 64              | 60        | 580  | 0.017 sec    | 0.024 sec   | 1.2 (Swap) | 1 GB   |
```

Reply:
We refer to it as Starter instance, as it's no longer Free with usage-based billing

Reply:
You probably have to run prettier again afterwards